lib/build_targets/package.dart

```diff
@@ -303,18 +292,38 @@ class NativeTpk {
       environment.logger
           .printStatus('The $securityProfile profile is used for signing.');
     }
-    result = await _processUtils.run(<String>[
+
+    // Run the native build
+    final String method =
+        "name: \"m1\", compiler:\"${tizenSdk.defaultNativeCompiler}\", extraoption: \"${extraOptions.join(' ').replaceAll('"', '\\"')}\", configs:[\"$buildConfig\"], rootstraps:[{name:\"${rootstrap.id}\", arch:\"${getTizenCliArch(buildInfo.targetArch)}\"}]";
+    final List<String> targets =
+        tizenProject.isMultiApp ? ['ui', 'service'] : ['.'];
+
+    final String build =
+        'name: "b1", methods: ["m1"], targets: ["${targets.join('","')}"]';
+    const String package = 'name: "test", targets:["b1"]';
+
+    final List<String> buildAppCommand = <String>[
       tizenSdk.tizenCli.path,
-      'package',
-      '-t',
-      'tpk',
+      'build-app',
+      '-m',
+      method,
+      '-b',
+      build,
+      '-p',
+      package,
       if (securityProfile != null) ...<String>['-s', securityProfile],
-      '--',
+      '-o',
       buildDir.path,
-    ]);
+      '--',
+      tizenDir.path,
```

Comment:
Ephemeral directories such as `Debug`, `lib`, and `res` should not be generated in the editable directory (`tizen/`).

Steps to fix:
1. Go to line 173 and rename the current `tizenDir` to something like `projectDir`.
1. Create a duplicate of `projectDir` named `tizenDir`:<p>
   ```dart
   final Directory tizenDir = tizenProject.editableDirectory;
   final Directory projectDir = tizenProject.editableDirectory;
   ```
1. Change the definition of `projectDir` to:<p>
   ```dart
   final Directory projectDir = tizenProject.isMultiApp ? tizenProject.uiAppDirectory : tizenDir;
   ```